README.md

```diff
@@ -1,11 +1,23 @@
 [![Build Status](https://travis-ci.com/brain-score/brain-score.web.svg?branch=master)](https://travis-ci.com/brain-score/brain-score.web)
 
 ## Setup
-Install dependencies: `pip install .`
+
+Create and activate a virtual environment
+
+Install dependencies: `pip install -r dev-requirements.txt`
```

Comment:
I don't know how you guys feel about this, it's an attempt to make the setup easier so there's not as much version wrangling. If there's a better way to do this or there's a reason to not do it at all, please let me know and I can remove it.

Reply:
I am not sure how we want to handle this, so I'll defer to @mschrimpf on this

Reply:
@mschrimpf sorry I'm just seeing this now! I didn't see a `requirements.txt` file and I didn't want to mess up any prod deploys which is why I took this route. I saw some yamls but I didn't know what was what so I was just trying to get a clean dev install, but yeah you're right ideally there should be one dependency file.

Reply:
yes agree, your call which one we want to keep

Reply:
I guess the real question is, how are you doing your dev installs? Should I be referencing one of the yaml files instead of this `requirements.txt` file I created? I see `environments.yml` but that doesn't list all of the dependencies or their versions and I'm not super familiar with all the possible incantations of `pip install`.

I don't do a lot of python work either so I'm just trying to document my steps to make things easier for the next dev to come along and get things set up. I will defer to you guys 100% in what you want here. I can rename the file `requirements.txt` I can remove the file entirely. Was just trying to do a little helpful house keeping since I was the latest to get things set up 🙂

Reply:
Let's do the following:
1. delete `environment.yml`
2. rename `dev-requirements.txt` -> `requirements.txt`
3. remove the [in-code requirements](https://github.com/brain-score/brain-score.web/blob/a09cc86dd96115a511af62278654f76dce09d5dd/setup.py#L10-L19) of the setup.py and instead read from `requirements.txt` (2)

Reply:
Based on this documentation https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/ I think it might be advisable to leave the list of dependencies in `setup.py`. Again, I'm not an expert on python dependency management so I'll defer to you on wtv you want here, but I found this while I was trying to figure out the "Right Way" to reference `requirements.txt` in the `setup.py` file.

Reply:
The best would be to move to `pyproject.toml` but that requires a bit more work.
How about we follow the [HuggingFace approach](https://github.com/huggingface/transformers/blob/bc4bbd9f6eb7a9281aa48bda737697435399b09f/setup.py#L97-L195) where they specify versions but by default remove them for the install?

Reply:
@kvfairchild says that migrating all brain-score repos to `pyproject.toml`s is on her near term todo list so she asked me to punt on this for now.